test/unit_test/deepdoc/parser/test_mistral_parser.py

```
@@ -0,0 +1,528 @@
+    monkeypatch.setattr(m.requests, "post", lambda url, headers=None, json=None, timeout=None, **kw: captured.update(json=json) or _Resp(200, {"pages": []}))
+    p._call_ocr(b"%PDF fake", "f.pdf", pages=None)
+    assert "pages" not in captured["json"]
+
+
+def test_call_ocr_raises_on_http_error(monkeypatch):
+    m = _load_mistral_parser(monkeypatch)
+    p = _make_parser(m, api_key="sk-bad")
+    monkeypatch.setattr(m.requests, "post", lambda *a, **k: _Resp(401, text="Unauthorized"))
+    try:
+        p._call_ocr(b"%PDF fake", "f.pdf", pages=None)
+        assert False, "expected RuntimeError"
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)